src/main/java/com/dragonballs/services/authentication/JwtAuthenticationResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.dragonballs.services.authentication;
+
+import java.io.Serializable;
+
+public class JwtAuthenticationResponse implements Serializable {
```

Comment:
Encapsualtion - no need for this class to know that it's holding JWT

Reply:
Could you elaborate, we shouldn't implement Serializable?

Reply:
I didn't mean that implements Serializable is wrong - even though it changes nothing :D

But - I don't really like JwtAuthenticationResponse naming, since it exposes type of token - JWT. I'd rename it to AuthenticationResponse